core/hosting/HostWithHostFxr/readme.md

```diff
@@ -0,0 +1,42 @@
+.NET Core Hosting Sample
+================
+
+This project demonstrates a way for a native process to host .NET Core using the `nethost` and `hostfxr` libraries. Documentation on the `nethost` and `hostfxr` APIs can be found [here](https://github.com/dotnet/core-setup/blob/master/Documentation/design-docs/native-hosting.md).
+
+Key Features
+------------
+
+Demonstrates how to locate and initialize .NET Core 3.0 from a non-.NET Core process and subsequently load and call into  a .NET Core assembly.
+
+The `nethost` header and library are part of the Microsoft.NETCore.DotNetAppHost package and are also installed as a runtime pack by the .NET SDK. The library should be deployed alongside the host. This sample uses the files installed with the .NET SDK. 
+
+The `coreclr_delegates.h` and `hostfxr.h` files are copied from the [core-setup](https://github.com/dotnet/core-setup) repo. 
+
+Addtional comments are contained in source and project files.
+
+Requirements
```

Comment:
perhaps name this "Prerequisites", to differentiate prerequisites from the two items that are also required?